HACKING.md

```diff
@@ -172,17 +172,20 @@ create a credentials config as a secret within OpenShift.
 First create a file with your secret content:
 
 ```
-cat <<'EOF' > /path/to/upload-secret
+mkdir dir
+cat <<'EOF' > dir/config
 [default]
 aws_access_key_id=keyid
 aws_secret_access_key=key
 EOF
+echo keyid > dir/accessKey
+echo key > dir/secretKey
```

Comment:
To an outside user the change here doesn't really make sense. Can we update the comment to say we're using the creds in multiple ways now? Or is there any way to pick up the creds from the secret we were creating in the past?

Reply:
Added a comment. I think it might work to just drop the config file and have cosa use the env vars, though I haven't tested that. Let's look at that in a follow-up?